FAIMS/FAIMS3
# Survey details main

## JIRA Ticket

Parent Jira Ticket : https://jira.csiro.au/browse/BSS-249
Subtaksk Tickets: 
- https://jira.csiro.au/browse/BSS-250
- https://jira.csiro.au/browse/BSS-252
- https://jira.csiro.au/browse/BSS-254
- https://jira.csiro.au/browse/BSS-253
- https://jira.csiro.au/browse/BSS-276
- https://jira.csiro.au/browse/BSS-277


## Description

This PR includes significant updates to the survey/notebook details section to align with the BSS design requirements. The layout structure, responsiveness, and styling have been enhanced, along with the addition of new features such as an archive survey section and improved refresh functionality. An edit button has also been added to enable content editing in the future.

## Proposed Changes

Layout Structure: Updated to match the new design specifications.
Responsiveness: Improved across various devices.
Styling: Applied consistent styles throughout the section.
Tabs: Extended the tabs as per the new design, records is equal to site,so haven't change that wording to 'sites'.
Details Tab Content: Updated to reflect the new layout.
Archive Survey Section: Added a new section for archiving surveys.
Refresh Notebooks Component: Enhanced with better content alignment and added an icon and button to refresh text.
Edit Button: Added to allow future content editing.
Title  - Positioning of Notebook/survey title to the center of screen for better visibility.
General styling updates

## How to Test

- Run the app
- From the list of notebooks, click on any notebook to navigate to the page where notebook details are shown
- You should be able to view the update tab layout, details tab layout specifically with all changes mentioned above.


## Checklist

- [ Y ] I have confirmed all commits have been signed.
- [ Y ] I have added JSDoc style comments to any new functions or classes.
- [ Y ] Relevant documentation such as READMEs, guides, and class comments are updated.
